hci/event/events/vendor_specific/hci_ext_reset_system_done.py

```diff
@@ -0,0 +1,17 @@
+from struct import unpack_from
+
+from hci.command.opcode import OpCode
+from .. import VendorSpecificEvent
+
+
+class HCI_EXT_ResetSystemDone(VendorSpecificEvent):
+    @property
+    def cmd_opcode(self):
```

Comment:
I think you called this just `opcode` in other events.

Reply:
in these event the documentation specify this paremeter as cmd_opcode